### ISSUE TITLE
Example for LLM .generated context.md

### ISSUE THREAD

**Issue by magaton** [Agentic-Insights/codebase-context-spec#16]
Hello, a very interesting idea and project!
Do you maybe have an example of a public Github repository for which you managed to generate a `.context.md` file programmatically (via LLM call with prompt)?

**Comment by killerapp** [comment]
Yes, there were some, I would take a look at the new generate option in the cli - I will eventually add support for using that cli to generate them with an LLM

**Comment by magaton** [comment]
Thanks, any reason why you removed a form for editing context.md? I found it pretty handy :)
